lib/components/molecules/Typeahead/tests/Typeahead.test.js

```diff
@@ -0,0 +1,44 @@
+import { shallowWithTheme } from '../../../../../__mock__/themeMock';
+
+import defaultConfig from '../Typeahead.mock';
+import ListItem from '../ListItem';
+import { TypeaheadVanilla } from '../index';
+
+describe('<Typeahead />', () => {
+  let TypeaheadComponent = '';
+  const onChange = jest.fn();
+
+  beforeEach(() => {
+    TypeaheadComponent = shallowWithTheme(
+      <TypeaheadVanilla
+        {...defaultConfig}
+        onChange={onChange}
+        renderSuggestion={item => <ListItem key={item}>{item}</ListItem>}
+      />
+    );
+  });
+
+  test('should render correctly', () => {
+    expect(TypeaheadComponent).toMatchSnapshot();
+    expect(TypeaheadComponent.exists()).toBe(true);
+    expect(TypeaheadComponent.exists({ type: 'search' })).toBe(true);
+  });
+
+  test('should fire on change', () => {
```

Comment:
You test are no where checking the data, if its getting rendered in intended places.